content/en/continuous_integration/explorer/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Continuous Integration Visibility Explorer
+kind: documentation
+description: Learn about the CI Visibility Explorer for test runs and pipeline executions.
+further_reading:
+  - link: "/continuous_integration/pipelines/"
+    tag: "Documentation"
+    text: "Explore pipeline data to resolve build problems"
+  - link: "/continuous_integration/tests/"
+    tag: "Documentation"
+    text: "Explore test data to find and fix problem tests"
+  - link: "https://www.datadoghq.com/blog/configure-pipeline-alerts-with-ci-monitors/"
+    tag: "Blog"
+    text: "Configure pipeline alerts with Datadog CI monitors"
+---
+
+## Overview
+
+The CI Visibility Explorer allows you to [search and filter](#search-and-filter), [visualize](#visualize), and [export](#export) test runs or pipeline executions at multiple levels using any tag.
```

Comment:
So you'll be able to say "CI Visibility Explorer" to reference either the Test Runs/Pipeline Executions view. The "Explorer" qualifies the unified search experience that's been rolled out to Tracing, Logs, Metrics, and RUM. 

The Error Tracking Explorer (https://docs.datadoghq.com/real_user_monitoring/error_tracking/explorer/) is not an actual Explorer—this one's fake because it doesn't allow users to group or analyze (create visualizations), so we would call this a "Search and Manage Errors" page instead of an "Error Tracking Explorer". Does that make sense?